docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -366,8 +366,56 @@ revisit this and keep track of dad as well as moms).
 
-Multiparity --- make twins with probability from ACS, about 2.5%.  See
-Section (16) for additional details.
+Multiparity --- make twins with probability 4%.  See Section (16) for
+additional details.
+
+Relationship -- the sim knows a parent-child dyad when the new
+simulant is initialized, and to come up with a consistent value for
+the "reference person" relationship field, we use the following
+mapping:
+
++--------------------------------------------------+----------------------------------------+
+| Parent relationship to reference person          | Child relationship to reference person |
++==================================================+========================================+
+| Reference Person                                 | Biological child                       |
++--------------------------------------------------+----------------------------------------+
+| Opposite-sex husband/wife/spouse                 | Biological child                       |
++--------------------------------------------------+----------------------------------------+
+| Opposite-sex unmarried partner                   | Biological child                       |
++--------------------------------------------------+----------------------------------------+
+| Same-sex husband/wife/spouse                     | Biological child                       |
++--------------------------------------------------+----------------------------------------+
+| Same-sex unmarried partner                       | Biological child                       |
++--------------------------------------------------+----------------------------------------+
+| Biological son or daughter                       | Grandchild                             |
++--------------------------------------------------+----------------------------------------+
+| Adopted son or daughter                          | Grandchild                             |
++--------------------------------------------------+----------------------------------------+
+| Stepson or stepdaughter                          | Grandchild                             |
++--------------------------------------------------+----------------------------------------+
+| Brother or sister                                | Other relative                         |
++--------------------------------------------------+----------------------------------------+
+| Father or mother                                 | Brother or sister                      |
++--------------------------------------------------+----------------------------------------+
+| Grandchild                                       | Other relative                         |
++--------------------------------------------------+----------------------------------------+
+| Parent-in-law                                    | Other nonrelative                      |
```

Comment:
did you want this as nonrelative or relative?

Reply:
other relative is better, thanks for catching that